rsearch/bin/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	search "github.com/romana/contrib/rsearch"
```

Comment:
IMO it's surprising to be aliasing your own package, since you have the option of naming it differently. No better suggestions, just pick one and use it consistently.